wallet-api/wallet-api.cabal

```diff
@@ -46,29 +63,42 @@ library
                  -Wredundant-constraints
     build-depends:
         base >=4.9 && <5,
-        ghc -any,
-        template-haskell -any,
-        language-plutus-core -any,
-        plutus-th -any,
-        containers -any,
-        free -any,
-        operational -any,
-        mtl -any,
-        transformers -any,
+        aeson -any,
+        aeson -any,
+        base64-bytestring -any,
+        bifunctors -any,
         bytestring -any,
-        text -any,
-        prettyprinter -any,
-        mmorph -any,
-        cryptonite -any,
         bytestring -any,
-        memory -any,
-        core-to-plc -any,
         cborg -any,
+        containers -any,
+        core-to-plc -any,
+        cryptonite -any,
+        errors -any,
+        free -any,
+        ghc -any,
         hedgehog -any,
-        microlens -any
+        language-plutus-core -any,
+        memory -any,
+        microlens -any,
+        mmorph -any,
+        monad-stm -any,
+        mtl -any,
+        natural-transformation -any,
+        operational -any,
+        plutus-th -any,
+        prettyprinter -any,
+        serialise -any,
+        servant -any,
+        servant-server -any,
+        stm -any,
+        template-haskell -any,
+        text -any,
+        text -any,
+        transformers -any,
+        unordered-containers -any
 
-    if (flag(development) && impl(ghc <8.4))
-        ghc-options: -Werror
+    -- if (flag(development) && impl(ghc <8.4))
+    --     ghc-options: -Werror
```

Comment:
In or out?

Reply:
'twas a lazy mistake, @michaelpj already reprimanded me

Reply:
:-D